sources/packages/backend/apps/api/src/services/disbursement-schedule/disbursement-schedule-service.ts

```diff
@@ -183,6 +192,7 @@ export class DisbursementScheduleService extends RecordDataModelService<Disburse
     const coeSortOptions = {
       applicationNumber: "application.applicationNumber",
       disbursementDate: "disbursementSchedule.disbursementDate",
+      studyEndDate: "offering.studyEndDate",
```

Comment:
For clarification, the sort options available at the DTO are as follows. Is the `coeStatus` missing?
```ts
@IsIn([
  "fullName",
  "applicationNumber",
  "disbursementDate",
  "coeStatus",
  "studyEndDate",
])
```